docs/user_settings.md

```diff
@@ -0,0 +1,31 @@
+
+<a name="UserSettings"></a>
+
+## UserSettings
+The UserSettings config provides a set of charts, themes and default values
+manages lifetime events/order execution. Internally it hosts a Manager
```

Comment:
This needs to be fixed :slightly_smiling_face:

Reply:
done